mcsema/Arch/Arch.cpp

```diff
@@ -331,6 +342,102 @@ size_t ArchDecodeInstruction(const uint8_t *bytes, const uint8_t *bytes_end,
   return total_size;
 }
 
+// Convert the given assembly instruction into an inline ASM operation in lieu
```

Comment:
Can you add to this comment an example of what the inline assembly looks like? E.g. here is instruct X, and here is the inline assembly produced. I think it would provide some context to understand how this code works.